aws_cloudwatch_policies/aws_cloudwatch_sensitive_loggroup_encrypted.py

```diff
@@ -0,0 +1,13 @@
+from fnmatch import fnmatch
+
+# replace the log groups ARN regex with the
+# log groups that should be encrypted with your CMK
+SENSITIVE_LOG_GROUP_ARN_REGEXS = {"*LogGroup-2*"}
+
+
+def policy(resource):
+    if SENSITIVE_LOG_GROUP_ARN_REGEXS and not any(
+        fnmatch(resource.get("Arn"), group_arn) for group_arn in SENSITIVE_LOG_GROUP_ARN_REGEXS
+    ):
+        return True
+    return resource["KmsKeyId"] is not None
```

Comment:
Maybe reverse the logic to check for encryption first, then if not encrypted, check if name is sensitive?

Reply:
I had figured kms key encryption is more rare to be on than off, but this is a good point - reversed.